<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbDialog</class>
 <widget class="QDialog" name="DbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание базы данных</string>
  </property>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
  </widget>
  <widget class="QPushButton" name="creatBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Укажите путь к рабочей директории</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileDialogBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
